kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -427,6 +430,18 @@ public static ConfigDef newConfigDef() {
           "S3 Proxy Password"
       );
 
+      configDef.define(
+          S3_PROXY_USE_EXPECT_CONTINUE_CONFIG,
+          Type.BOOLEAN,
+          S3_PROXY_USE_EXPECT_CONTINUE_DEFAULT,
+          Importance.LOW,
+          "Sets if use expect continue should be enabled when accessing S3 through a proxy.",
```

Comment:
This setting has implications. 

We might want to include info that is more related to this comment: 
https://github.com/aws/aws-sdk-java/blob/master/aws-java-sdk-core/src/main/java/com/amazonaws/ClientConfiguration.java#L326-L336

Rather what the javadoc for the method says. At least closer to what the first paragraph of this link ^^ says. 
Also, fyi, the http header is "expect continue". The word "use" is related more to the method in AWS SDK.